### DISCUSSION TITLE
Edit multiple animations from a separate editing and export them in a single spritesheet

### DISCUSSION THREAD

**Discussion by merpath** [Orama-Interactive/Pixelorama#951] [Q&A]
Is it possible to do that?

**Comment by OverloadedOrama** [comment]
Yes, see https://www.oramainteractive.com/Pixelorama-Docs/user_manual/save_and_export#export-a-spritesheet

**Comment by merpath** [comment]
I know that way, what I mean is to export to one spritesheet from different animations like run, idle, jump etc from a separate editing. Is it possible?
Currently what I know as the best way is to edit them as one animation and export them with some split in the middle to separate them (just for convenience).

![image](https://github.com/Orama-Interactive/Pixelorama/assets/105505135/a3076d13-6242-4dfb-a6c7-fa2ebbc9378b)

I've edited my title btw.

**Reply by Variable-ind** [reply]
You mean export each tag as a separate Sprite sheet?
Or one sprite sheet having each tag having it's separate row?

**Reply by merpath** [reply]
each tag on separate row
